lessons/bayes_theorem.ipynb

```diff
@@ -734,7 +734,6 @@
    "cell_type": "code",
```

Comment:
It's really annoying that the PR is cluttered with files where only the metadata has changed due to saving the notebook.

Reply:
I feel your pain.